src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/manufacturer.yml

```diff
@@ -38,9 +38,6 @@ admin_manufacturers_edit:
   defaults:
     _controller: 'PrestaShopBundle:Admin/Sell/Catalog/Manufacturer:edit'
     _legacy_controller: AdminManufacturers
-    _legacy_link: AdminManufacturers:updatemanufacturer
```

Comment:
Why this ?

Reply:
@matks its just because all those legacy links where removed previously and this one left alone. So now i can add them all back again, or leave it for last PR( for enabling the whole page)

Reply:
All right 👍